compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/PositionUtil.java

```diff
@@ -109,18 +108,17 @@ static boolean isPosWithinRange(LinePosition position, LineRange enclRange) {
                 && (posLine == enclEndLine && posOffset <= enclEndOffset || posLine < enclEndLine);
     }
 
-    static boolean isWithinParenthesis(LinePosition linePosition, Token openParen, Token closedParen) {
+    static boolean isWithinParenthesis(LinePosition linePosition, LineRange openParenLineRange,
```

Comment:
We can generalize this more (like `isWithinTokens()`- to check whether the line position is within two tokens). However, I'm +1 for this name as well. I just wanted to bring some ideas.